src/viem/getLockableWallet.ts

```diff
@@ -51,6 +64,29 @@ export default function getLockableViemWallet(
       // For instance we will later need signTransaction which we can add here by
       // importing the signTransaction action and blocking it with the checkLock function
       // Introduction to wallet actions: https://viem.sh/docs/actions/wallet/introduction.html
+      prepareTransactionRequest: (args) => {
+        checkLock()
+        return prepareTransactionRequest(
+          client,
+          args
+        ) as Promise<PrepareTransactionRequestReturnType>
+      },
```

Comment:
Are we using this one in this PR?
Since it doesn't yet correctly work for Celo, I'd prefer we don't include it for now and leave a note about it?